Application/EdFi.Ods.Features/ExternalCache/ExternalCacheProvider.cs

```diff
@@ -169,7 +169,8 @@ private object Deserialize(string @string)
 
             try
             {
-                return JsonConvert.DeserializeObject(@string, _defaultSerializerSettings);
+                // JsonConvert.DeserializeObject without a Type, returns a JObject, that will fail to be casted to a DescriptorMaps object
```

Comment:
JsonConvert.DeserializeObject returns a JObject that will throw an exception when trying to cast it to DescriptorMaps.

Reply:
How is this change rationalized against the comment on [Line 90](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/765/files#diff-f532e1bea12f11735b35bffb23c070c38888d4823ed590b96deebc861bb7a70bR90) above? What has this changed?

Reply:
ExternalCacheProvider is used in 3 overrides

PersonUniqueIdToUsiCache -> deserializes to IdentityValueMaps
ContextualCachingInterceptor -> registered to intercept IDescriptorMapsProvider, deserializes to DescriptorMaps
CachingApiClientDetailsProviderDecorator -> deserializes to ApiClientDetails